### PR TITLE
[Merged by Bors] - feat: custom action step (PL-146)

### PR DESCRIPTION
**Pull request by trs**
**Fixes or implements PL-146**

### Related PRs

- https://github.com/voiceflow/general-service/pull/399

### Checklist

- [ ] this is a breaking change and should publish a new major version
- [ ] appropriate tests have been written

### CODE REVIEW

File: packages/base-types/src/node/customAction.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { NodeType } from './constants';
+import { BaseNoMatchStepData, BaseNoReplyStepData, BaseStep, BaseStepPorts, BuiltInNextPort, BuiltInNoMatchNoReplyPorts } from './utils';
+
+export interface StepBuiltInPorts extends BuiltInNextPort, BuiltInNoMatchNoReplyPorts {}
+
+export interface StepPorts extends BaseStepPorts<StepBuiltInPorts> {}
+
+export interface StepData extends BaseNoMatchStepData, BaseNoReplyStepData {
```
Comment (trs) [inline packages/base-types/src/node/customAction.ts:8]: This should be the "pointer". Is it a step or node?

**Comment by zhihil** [comment]
(^) Just forced pushed to rebase to latest `master`. I have a branch that's forked off of yours, so I had to rebase in this order:

`master` -> this one -> mine

**Review by zhihil — APPROVED** [review]
Seems ok

**Pull request closed without merging**